### PR TITLE
Wagtail 5.1: Shared include field_as_li.html will be removed

### PR DESCRIPTION
**Pull request by katdom13**
[Wagtail 5.1 release notes](https://docs.wagtail.org/en/stable/releases/5.1.html)

Changes:
- [Shared include field_as_li.html will be removed](https://docs.wagtail.org/en/stable/releases/5.1.html#shared-include-field-as-li-html-will-be-removed)

Upstream PR: https://github.com/torchbox/wagtail-purge/pull/7

### CODE REVIEW

**Review by nickmoreton — CHANGES_REQUESTED** [review]
Hi @katdom13 Can you include a not in the Changelog please.

File: wagtail_purge/templates/index.html [hunk 1]
```diff
@@ -24,7 +24,9 @@
         {% csrf_token %}
         <ul class="fields">
             {% for field in form %}
-            {% include "wagtailadmin/shared/field_as_li.html" with field=field %}
+            <li>
+                {% include "wagtailadmin/shared/field.html" with field=field %}
```
Comment (nickmoreton) [inline wagtail_purge/templates/index.html:28]: Nice find, it's a hard one to spot 👍

**Comment by katdom13** [comment]
Hi @nickmoreton ,
How about this entry?
https://github.com/torchbox-forks/wagtail-purge/pull/1/commits/29b4a10d207ed3b654430284bbed8168170898f2

**Review by nickmoreton — APPROVED** [review]
Hi @katdom13 I approve this. Thanks

**Pull request merged**